package.json

```diff
@@ -22,32 +22,32 @@
         "sulu-website-bundle": "file:vendor/sulu/sulu/src/Sulu/Bundle/WebsiteBundle/Resources/js"
     },
     "devDependencies": {
-        "autoprefixer": "^7.2.5",
+        "autoprefixer": "^9.5.1",
         "babel-core": "^6.25.0",
         "babel-loader": "^7.1.1",
         "babel-plugin-transform-class-properties": "^6.24.1",
         "babel-plugin-transform-decorators-legacy": "^1.3.4",
         "babel-plugin-transform-object-rest-spread": "^6.23.0",
         "babel-preset-env": "^1.6.1",
         "babel-preset-react": "^6.24.1",
-        "clean-webpack-plugin": "^1.0.0",
+        "clean-webpack-plugin": "^2.0.1",
         "core-js": "^2.5.1",
-        "css-loader": "^0.28.4",
-        "file-loader": "^1.1.11",
+        "css-loader": "^1.0.1",
+        "file-loader": "^3.0.1",
         "glob": "^7.1.2",
-        "mini-css-extract-plugin": "^0.4.0",
-        "postcss-calc": "^6.0.1",
+        "mini-css-extract-plugin": "^0.6.0",
+        "postcss-calc": "^7.0.1",
         "postcss-hexrgba": "^1.0.0",
-        "postcss-import": "^10.0.0",
+        "postcss-import": "^12.0.1",
         "postcss-loader": "^2.0.6",
```

Comment:
There would also be a new major release, doesn't that work? In general, how did you choose which packages to update?

Reply:
Could not get postcss-loader upgrade and css-loader upgrade to work and some others need a babel update.